Dockerfile.builder

```diff
@@ -55,7 +56,8 @@ RUN cd /tmp && \
     ln -s /usr/local/yarn/bin/yarn /usr/local/bin/yarn
 
 # Install Chrome for installer gui tests
-RUN wget --quiet -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
+RUN test "$(go env GOARCH)" = amd64 || exit 0; \
```

Comment:
Can the test suite use chromium and provide reliable test results? We could install it at least for non-amd64 images.